testdata/p4_16_samples_outputs/issue1670-bmv2.p4.p4info.txt

```diff
@@ -0,0 +1,3 @@
+pkg_info {
```

Comment:
does it make sense to have a p4info file if the test is in `P4RUNTIME_EXCLUDE`, how did the file even get generated?